sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 7.10.0-beta.1 (Unreleased)
 
 ### Features Added
+- Expose `ServiceBusClientBuilder.customEndpointAddress` to support connecting to an intermediary before Azure ServiceBus.
```

Comment:
Follow the convention in the file. `Supported` etc.

Reply:
Sure, I'll change it.